powerplantmatching/data.py

```diff
@@ -1582,6 +1582,256 @@ def IRENASTAT(raw=False, update=False, config=None):
     return df
 
 
+def GCPT(raw=False, update=False, config=None):
+    """
+    Importer for the global coal powerplant tracker from global energy monitor.
+
+    Parameters
+    ----------
+    raw : boolean, default False
+        Whether to return the original dataset
+    update: bool, default False
+        Whether to update the data from the url.
+    config : dict, default None
+        Add custom specific configuration,
+        e.g. powerplantmatching.config.get_config(target_countries='Italy'),
+        defaults to powerplantmatching.config.get_config()
+    """
+
+    config = get_config() if config is None else config
+    fn = get_raw_file("GCPT", update=update, config=config)
+    df = pd.read_csv(fn)
+
+    if raw:
+        return df
+
+    RENAME_COLUMNS = {
+        "Plant": "Name",
+        "Combustion technology": "Technology",
+        "Coal type": "Fueltype",
+        "Capacity (MW)": "Capacity",
+        "Latitude": "lat",
+        "Longitude": "lon",
+        "Year": "DateIn",
+        "RETIRED": "DateOut",
+        "Tracker ID": "projectID",
+    }
+    fueltype_dict = {
+        "bituminous": "Hard Coal",
+        "lignite": "Lignite",
+        "unknown": "Hard Coal",
+        "sub-bit": "Hard Coal",
+        "bituminous/sub-bit": "Hard Coal",
+        "wstbituminous": "Hard Coal",
+        "unknown(ccs90)": "Hard Coal",
+        "hard": "Hard Coal",
+        "anth": "Hard Coal",
+        "lignite(ccs90)": "Lignite",
+        "sub-bit(ccs90)": "Hard Coal",
+        "lignite/sub-bit": "Lignite",
+        "wstcoal": "Hard Coal",
+        "bituminous(ccs90)": "Hard Coal",
+        "bituminous(ccs30)": "Hard Coal",
+        "lignite/bituminous": "Lignite",
+        "anth/bituminous": "Hard Coal",
+        "anth/culm": "Hard Coal",
+        "bituminous/wstbituminous": "Hard Coal",
+    }
+    df = df.rename(columns=RENAME_COLUMNS)
+    df_final = (
+        df.pipe(clean_name)
+        .pipe(set_column_name, "GCPT")
+        .pipe(convert_to_short_name)
+        .dropna(subset="Capacity")
+        .pipe(lambda x: x.replace({"Fueltype": fueltype_dict}))
+        .pipe(lambda x: x.assign(Technology="Steam Turbine"))
+        .pipe(lambda x: x.assign(Set="PP"))
+        .assign(
+            DateIn=df["DateIn"].apply(pd.to_numeric, errors="coerce"),
+            DateOut=df["DateOut"].apply(pd.to_numeric, errors="coerce"),
+            lat=df["lat"].apply(pd.to_numeric, errors="coerce"),
+            lon=df["lon"].apply(pd.to_numeric, errors="coerce"),
+        )
+        .pipe(lambda x: x[df.columns.intersection(config.get("target_columns"))])
+        .pipe(config_filter, config)
+    )
+
+    return df_final
+
+
+def GGtPT(raw=False, update=False, config=None):
```

Comment:
I see why you named it "GGtPT", but what do you think about renaming it "GGTPT" instead? 
Would require changes across the PR wherever GGtPT was mentioned

Reply:
Main reason I went with GGtPT was to stay consistent with how GEM call their datasets. See here https://globalenergymonitor.org/projects/global-geothermal-power-tracker/ . I would suggest we stay with their naming conventions.